tests/gocase/integration/cli/cli_test.go

```diff
@@ -38,14 +39,16 @@ import (
 
 const defaultByteBufLen = math.MaxUint16
 
+var cliPath = flag.String("cliPath", "redis-cli", "path to redis-cli")
```

Comment:
As mentioned here https://github.com/apache/incubator-kvrocks/pull/1012#discussion_r999529162, I think `go test` will fail if `cliPath` is passed explicitly, e.g.
```
cd tests/gocase
go test ./... -cliPath=something
```

Reply:
Oops, I misunderstand the rule here. Reverting...